config.yaml

```diff
@@ -6,6 +6,7 @@ parent_calls_tracked: 10
 REDIS_URI: 'redis://localhost:6379/5'
 CRAWLER_GET_TIMEOUT: 180
 CRAWLER_HEAD_TIMEOUT: 10
+KNOWN_HEAD_OFFENDER_DOMAINS: []
```

Comment:
Not super happy with the setting name, maybe `HEAD_DOMAINS_BLACKLIST` or something in this vein?

Reply:
Ah ah ;-) Yours is more consensual, I'll give you that :p